heron/common/src/python/utils/misc/communicator.py

```diff
@@ -12,12 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 '''communicator.py: module responsible for communication between Python heron modules'''
+import sys
```

Comment:
@kramasamy Only reason why they are here is because equivalent ones in Java are under ``heron/common``.